lib/toml/toml.go

```diff
@@ -0,0 +1,60 @@
+package toml
```

Comment:
There is a duplicate file in automate-deployment/pkg/toml, can we remove that file ?

Reply:
that is used in some other places, removal of that should be a Refactoring Work, not to be mixed with Feature work, to avoid test area expansion and scope of bugs. Please feel free to create separate PR for that.